code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -539,6 +539,12 @@
 	if (!M) return
 	if (isflockmob(M)) return
 	if (!isdead(src) && src.flock)
+		if (!src.controller && prob(25) && (src in oviewers(M)))
```

Comment:
```suggestion
		if (src.is_npc && prob(25) && (src in oviewers(M)))
```
`is_npc` reflects controller posession as well as dormant state, probably better to use